styles.css

```diff
@@ -424,3 +431,303 @@ footer hr {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #work-left {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-width: 60%;
+  }
+
+  .work img {
+    height: 100%;
+    width: 90%;
+  }
+
+  #work-right {
+    display: flex;
+    flex-direction: column;
+  }
+
+  #work-right h2 {
+    margin-top: 0%;
+    padding-bottom: 0%;
+    margin-bottom: 3%;
+  }
+
+  #work-right p {
+    padding-top: 0%;
+    margin-top: 0%;
+  }
+
+  #labels ul {
+    display: flex;
+    flex-wrap: nowrap;
+    margin-right: 1%;
+    padding-right: 1%;
+  }
+
+  #labels ul li {
+    margin-right: 2%;
+  }
+
+  .card {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 2%;
+    height: 100%;
+  }
+
+  .card-body-head {
+    display: flex;
+    align-content: flex-end;
+    padding-top: 50%;
+
+    /* padding-bottom: 2%; */
+  }
+
+  .card-body-head h3 {
+    width: 100%;
+    height: 15vh;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .card1 .card-body-head {
+    display: none;
+  }
+
+  .card1 {
+    background-image: url("Images/card1.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .card2 button {
+    display: none;
+  }
+
+  .card2 {
+    background-image: url("Images/card2.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .card3 button {
+    display: none;
+  }
+
+  .card3 {
+    background-image: url("Images/card3.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .card4 button {
+    display: none;
+  }
+
+  .card4 {
+    background-image: url("Images/card4.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .card5 button {
+    display: none;
+  }
+
+  .card5 {
+    background-image: url("Images/card2.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .card6 button {
+    display: none;
+  }
+
+  .card6 {
+    background-image: url("Images/card3.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .about {
+    padding-left: 10%;
+    background-image: url("Images/about-desktop.png");
+    background-size: 40% 40%;
+    background-repeat: no-repeat;
+    background-position: right center;
+    height: 70%;
+  }
+
+  #aboutme {
+    width: 80%;
+    padding-bottom: 15%;
+  }
+
+  .center {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-left: 10%;
+    padding-right: 10%;
+    text-align: left;
+  }
+
+  #skills-ul {
+    padding-left: 0%;
+  }
+
+  .footer-top {
+    display: flex;
+    flex-direction: row;
+  }
+
+  form {
+    padding-top: 5%;
+    width: 70%;
+  }
+}
```

Comment:
- [ ] Your form margin/padding is not symmetrical. Kindly use the same dimensions the Figma template provides.
![image](https://user-images.githubusercontent.com/61919617/171908231-9af5f93a-a06f-4d38-a015-015d114adc56.png)
![image](https://user-images.githubusercontent.com/61919617/171908308-5f212d77-7a7b-4df9-9bd5-27e5b0a84e0c.png)